### PR TITLE
Some simple changes in the main README and the CLI tutorial, mainly typos

### PR DESCRIPTION
**Pull request by mvanbrab**

### CODE REVIEW

File: Tutorial-cli.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ This tutorial only regards the CLI interface of the Bashlib library. For the Nod
 All aliases are calculated from the root of the cloned bashlib repo.
   - bashlib-css  - `alias bashlib-css="node bashlib/css/bin/css.js"`
   - bashlib-solid  - `alias bashlib-solid="node bashlib/solid/bin/solid.js"`
-  - bashlib-auth - `alias bashlib-auth="node bashlib/solid/bin/solid.js --auth interactive --idp <your pod identity provider>"`
+  - bashlib-auth - `alias bashlib-auth="node bashlib/solid/bin/solid.js --auth token -t .tokens/.bobs-auth-token"`
```
Comment (mielvds) [inline Tutorial-cli.md:10]: @mvanbrab why is this wrong?

File: Tutorial-cli.md [hunk 2]
```diff
@@ -377,7 +377,7 @@ The `perms` command provides three operations to list, edit and delete permissio
 
 To list the permissions of your profile resource, we can use the following command:
 ```
-bashlib-auth perms list webid:https://github.com/CommunitySolidServer/CommunitySolidServer
+bashlib-auth perms list webid:
```
Comment (mielvds) [inline Tutorial-cli.md:380]: @mvanbrab an empty prefix seems wrong?

**Review by mielvds — CHANGES_REQUESTED** [review]

File: Tutorial-cli.md (hunk 1 above)
Comment (mvanbrab) [reply Tutorial-cli.md:10]: It's not wrong, but it's not consistent with the redefinition of this alias on line 208, hence confusing for someone following the tutorial and making aliases here already.

File: Tutorial-cli.md (hunk 2 above)
Comment (mvanbrab) [reply Tutorial-cli.md:380]: The thing after webid: was a copy-paste error (the CSS github repo by the way). Checked this change. The updated command worked. Original did not. Author agreed by the way :-)

**Pull request merged**